barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover.hpp

```diff
@@ -432,14 +542,17 @@ template <class ProverInstances_> class ProtoGalaxyProver_ {
     {
         size_t param_idx = 0;
         auto to_fold = instances.relation_parameters.get_to_fold();
-        for (auto& folded_parameter : to_fold) {
+        auto to_fold_optimised = instances.optimised_relation_parameters.get_to_fold();
+        for (auto [folded_parameter, optimised_folded_parameter] : zip_view(to_fold, to_fold_optimised)) {
             Univariate<FF, ProverInstances::NUM> tmp(0);
             size_t instance_idx = 0;
             for (auto& instance : instances) {
                 tmp.value_at(instance_idx) = instance->relation_parameters.get_to_fold()[param_idx];
                 instance_idx++;
             }
             folded_parameter = tmp.template extend_to<ProverInstances::EXTENDED_LENGTH>();
+            optimised_folded_parameter =
```

Comment:
can we update the PG unit tests to test what should be in the optimised folded parameters and highlight the differences?